aviary/interface/methods_for_level2.py

```diff
@@ -2436,46 +2436,31 @@ def _add_gasp_landing_systems(self):
-        if reserves_opt == Reserves_Type.Set_Direct:
-            if reserves_val > 10:
-                self.model.add_subsystem(
-                    "reserves_calc",
-                    om.ExecComp(
-                        f"reserve_fuel = {reserves_val}",
-                        reserve_fuel={"val": reserves_val, "units": "lbm"}
-                    ),
-                    promotes_outputs=[("reserve_fuel", reserves_name)],
-                )
-            else:
-                raise ValueError(f'"{Aircraft.Design.RESERVES}" is not valid below 10.')
-        elif reserves_opt == Reserves_Type.Set_Fraction:
-            if reserves_fac <= 0 and reserves_fac >= -1:
-                reserves_fac = -reserves_fac
-                self.model.add_subsystem(
-                    "reserves_calc",
-                    om.ExecComp(
-                        f"reserve_fuel = {reserves_fac}*(takeoff_mass - final_mass)",
-                        takeoff_mass={"units": "lbm"},
-                        final_mass={"units": "lbm"},
-                        reserve_fuel={"units": "lbm"}
-                    ),
-                    promotes_inputs=[
-                        ("takeoff_mass", Mission.Summary.GROSS_MASS),
-                        ("final_mass", Mission.Landing.TOUCHDOWN_MASS),
-                    ],
-                    promotes_outputs=[("reserve_fuel", reserves_name)],
-                )
-            elif reserves_fac > 0 and reserves_fac <= 1:
-                raise ValueError(
-                    f'"{Aircraft.Design.RESERVES_FRACTION}" between 0 and 1 is not implemented.')
-            else:
-                raise ValueError(
-                    f'"{Aircraft.Design.RESERVES_FRACTION}" is not valid if less than -1 or larger than 1.')
-        else:
-            raise ValueError(f'"{Aircraft.Design.RESERVES_OPTION}" must be \
-                             {Reserves_Type.Set_Directn} or {Reserves_Type.Set_Fraction}.')
+        if reserves_val > 0.0:
+            self.model.add_subsystem(
+                "reserves_calc",
+                om.ExecComp(
+                    f"reserve_fuel = {reserves_val}",
+                    reserve_fuel={"val": reserves_val, "units": "lbm"}
+                ),
+                promotes_outputs=[("reserve_fuel", reserves_name)],
+            )
+        elif reserves_frac >= 0:
+            self.model.add_subsystem(
+                "reserves_calc",
+                om.ExecComp(
+                    f"reserve_fuel = {reserves_frac}*(takeoff_mass - final_mass)",
+                    takeoff_mass={"units": "lbm"},
+                    final_mass={"units": "lbm"},
+                    reserve_fuel={"units": "lbm"}
+                ),
+                promotes_inputs=[
+                    ("takeoff_mass", Mission.Summary.GROSS_MASS),
+                    ("final_mass", Mission.Landing.TOUCHDOWN_MASS),
+                ],
+                promotes_outputs=[("reserve_fuel", reserves_name)],
+            )
```

Comment:
```suggestion
	reserves_opt =  self.aviary_inputs.get_val(Aircraft.Design.RESERVES_OPTION)
        if reserves_opt is Reserves_Type.Set_Direct:
            self.model.add_subsystem(
                "reserves_calc",
                om.ExecComp(
                    f"reserve_fuel = {reserves_val}",
                    reserve_fuel={"val": reserves_val, "units": "lbm"}
                ),
                promotes_outputs=[("reserve_fuel", reserves_name)],
            )
        elif reserves_opt is Reserves_Type.Set_Fraction:
            self.model.add_subsystem(
                "reserves_calc",
                om.ExecComp(
                    f"reserve_fuel = {reserves_frac}*(takeoff_mass - final_mass)",
                    takeoff_mass={"units": "lbm"},
                    final_mass={"units": "lbm"},
                    reserve_fuel={"units": "lbm"}
                ),
                promotes_inputs=[
                    ("takeoff_mass", Mission.Summary.GROSS_MASS),
                    ("final_mass", Mission.Landing.TOUCHDOWN_MASS),
                ],
                promotes_outputs=[("reserve_fuel", reserves_name)],
            )
```

Reply:
I haven't decided if I will use `Aircraft.Design.RESERVES_OPTION`. So, I will hold on this suggestion.